WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductFromImage/AddProductFromImageViewModel.swift

```diff
@@ -165,17 +165,17 @@ private extension AddProductFromImageViewModel {
                 [nameViewModel, descriptionViewModel].forEach { $0.reset() }
                 generateProductDetails()
             } catch {
+                // Reset scanned texts and generated content from previous image
+                scannedTexts = []
+                [nameViewModel, descriptionViewModel].forEach { $0.reset() }
+
```

Comment:
❓ Is there any reason why this is not reset at the beginning of the method instead? I thought we'd want to reset the states when a new image comes in.

Reply:
Good point. I didn't think of this. Done in https://github.com/woocommerce/woocommerce-ios/pull/10297/commits/8465b078080b639969425de6f31794f20425b277